src/components/Common/OpenPositions.js

```diff
@@ -41,26 +39,20 @@ const renderJobsForlocation = (jobs, location, key) => {
   )
 }
 
-const OpenPositions = React.forwardRef(
-  ({ data: { title, getInTouchText, getInTouchTitle } }, ref) => (
-    <div ref={ref}>
-      <Section greyBg id="open-positions">
-        <Padding top={{ smallPhone: 3, tablet: 4 }}>
-          <SectionTitle>{title}</SectionTitle>
-          <JobsByLocation>
-            {jobs =>
-              jobs.map(({ location, jobs: jobsForLocation }, idx) =>
-                renderJobsForlocation(jobsForLocation, location, idx)
-              )
-            }
-          </JobsByLocation>
-          <Padding top={{ smallPhone: 3, tablet: 4 }}>
-            <Hr />
-          </Padding>
-        </Padding>
-      </Section>
-      <GetInTouch title={getInTouchTitle} contactText={getInTouchText} />
-    </div>
-  )
-)
+const OpenPositions = React.forwardRef(({ data: { title } }, ref = null) => (
+  <div ref={ref}>
+    <Section greyBg id="open-positions">
+      <Padding top={{ smallPhone: 3, tablet: 4 }}>
```

Comment:
As Anna said in a comment, please remove Padding here.

The reason for removing padding in instances like this is because we're adding HTML to the DOM that isn't necessary. Because we have this wrapper `Section` we can use styled components to add padding without introducing more HTML.

```
const StyledSection = styled(Section)`
   /* all the padding here with breakpoints included */
`
```

This way you go from:
```
<div class="section">
    <div class="padding-element">
        <h2>Title</h2>
    </div>
</div>
```

To:
```
<div class="section">
    <h2>Title</h2>
</div>
```

It might not seem like much but do this 100+ times and you're adding more in general which is not what we want.

As I said "in instances like this", there are instances when Padding might be ok to use. For example say you had something like this: 

```
<div>
    <h2>Title</h2>
    <p>copy section</p>
    <img src="" alt="">
</div>
```

and needed padding on this div and only padding, that might be a decent time to use it but ultimately I don't see the advantage of using `Padding` for a site like ours.

@AniaMakes